### PR TITLE
Feat(community partners): add CAREamics to json

### PR DESCRIPTION
**Pull request by melisande-c**
This PR adds CAREamics' info to the `bioimageio_collection_config.json` file, so we can be added as a community partner, following #100.

### Notes

@FynnBe, for the github field in the contact info, I put the CAREamics maintainers team, but I noticed everyone else has a user name. Let me know if this is not compatible and I will change it to individual users.

@jdeschamps do you mind having a look over this to make sure the details are correct, thanks.

### CODE REVIEW

**Review by FynnBe — COMMENTED** [review]
LGTM, @cfusterbarcelo @oeway can maybe state the requirements for the `contact` field... AFAIK it is only used in the docs by an imjoy app. As far as I am concerned there are no special requirements.

File: bioimageio_collection_config.json [hunk 1]
```diff
@@ -506,6 +506,34 @@
...
+                            ],
+                            "email": "rse@fht.org",
+                            "github": "https://github.com/orgs/CAREamics/teams/maintainers",
+                            "name": "CAREamics Maintainers"
+                        }
+                    ],
+                    "default_type": "model",
+                    "docs": "https://careamics.github.io/",
+                    "explore_button_text": "Start Exploring",
+                    "icon": "https://github.com/CAREamics/.github/blob/main/profile/images/logo_careamics_128.png",
```
Comment (jdeschamps) [inline bioimageio_collection_config.json:528]: Shouldn't the link be to the raw file https://raw.githubusercontent.com/CAREamics/.github/refs/heads/main/profile/images/logo_careamics_128.png?

**Review by jdeschamps — APPROVED** [review]

**Comment by jdeschamps** [comment]
I only have one comment! Thanks @melisande-c !!

File: bioimageio_collection_config.json (hunk 1 above)
Comment (melisande-c) [reply bioimageio_collection_config.json:528]: oops, thanks, will update

**Comment by cfusterbarcelo** [comment]
Hi, yes the contact field is used in the docs to give some contact details about each CP. It is good to have it but not mandatory.

**Pull request merged**